catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -119,15 +119,15 @@ private void extractPdfMetadata(PDDocument pdfDocument, MetacardImpl metacard) {
         if (creationDate != null) {
             metacard.setCreatedDate(creationDate.getTime());
             addXmlElement("creationDate", DATE_FORMAT.format(creationDate), metadataField);
+            LOGGER.info("PDF Creation date was: {}", DATE_FORMAT.format(creationDate));
```

Comment:
nit - these log statements seem more like debug info to me.  See https://codice.atlassian.net/wiki/display/DDF/Log+Levels.